### PR TITLE
Maps FiberImplant and Headframes 2019

### LINKED ISSUE #46: Map 2019 sharepoint database for Fiber Implant
# User story
As a user, I want to map the 2019 sharepoint database to the aind-data-schema FiberImplant model, so I can have a more complete Procedures instance

# Acceptance criteria
- [x] Map all known procedure fields
- [x] Map all known fields for OphysProbe (stereotactic_coordinate_ml and _ap, stereotactic_coordinate_dv, TBD)
- [x] Create a list of Ophysprobes for FiberImplant instance
- [x] handle validation errors

# Sprint Ready Checklist 
- [x] 1. Acceptance criteria defined
- [x] 2. Team understands acceptance criteria
- [x] 3. Team has defined solution / steps to satisfy acceptance criteria
- [x] 4. Acceptance criteria is verifiable / testable
- [x] 5. External / 3rd Party dependencies identified
- [x] 6. Ticket is prioritized and sized

# Notes
missing these fields: name, manufacturer, part_number, core_diameter, numerical_aperture, ferrule_material

### PR DESCRIPTION
**Pull request by mekhlakapoor**
closes #46 #48 #50

### CODE REVIEW

**Review by dyf — APPROVED** [review]

**Review by jtyoung84 — APPROVED** [review]
This looks okay to me. In the future, try to break large pull requests into smaller ones if possible.

File: tests/sharepoint/resources/list_item2.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   "Weight_x0020_after_x0020_Surgery": null,
   "PedigreeName": "Slc6a3-Cre-650102",
   "Breg2Lamb": null,
-  "HeadpostType": "AI Straight Headbar",
+  "HeadpostType": "CAM-style headframe (0160-100-10 Rev A)",
```
Comment (saskiad) [inline tests/sharepoint/resources/list_item2.json:35]: shouldn't this be "CAM-style" ? I might not understand what this particular file is - but doesn't the client break this into type and part number?

File: tests/sharepoint/resources/list_item3.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "Weight_x0020_after_x0020_Surgery": null,
   "PedigreeName": "Slc6a3-Cre-650102",
   "Breg2Lamb": null,
-  "HeadpostType": "AI Straight Headbar",
+  "HeadpostType": "Mesoscope-style well with NGC-style headframe (0160-200-20/0160-100-10)",
```
Comment (saskiad) [inline tests/sharepoint/resources/list_item3.json:35]: same question - doesn't type and part number get separated?

Comment (saskiad) [reply tests/sharepoint/resources/list_item3.json:35]: oh this is the actual data from the SWR rather than the json is creates - is that right?

**Review by saskiad — APPROVED** [review]
I think it looks good - I was confused by some of the files that I think are examples - you can see my comments - but I think I figured out that they were the data from the SWR and not the parsing by the client. If that's not the case, then let's fix that.

**Pull request merged**